src/test/java/foodwhere/testutil/ReviewBuilder.java

```diff
@@ -70,7 +69,7 @@ public ReviewBuilder withContent(String content) {
     }
 
     public Review build() {
-        return new Review(new StallName(name.fullName), date, content, details);
+        return new Review(new Name(name.fullName), date, content, details);
```

Comment:
I think `name` would be more appropriate here since it is already of type `Name` - this was a relic of two name classes and I tried to use `Name` as much as possible in the code previously without touching the `Review` class.